backend/attendance-service/src/main/java/org/egov/repository/querybuilder/RegisterQueryBuilder.java

```diff
@@ -6,7 +6,7 @@
 
 import java.util.List;
 @Component
-public class AttendanceRegisterQueryBuilder {
+public class RegisterQueryBuilder {
```

Comment:
Have the queries been written as per our last discussion, with 4 fundamental queries and single join query? 
First of all, is this task part of this PR or anitha's PR?